gemfile.d/lock.rb

```diff
@@ -0,0 +1,6 @@
+case ENV['FOREMAN_VERSION']
+when '3.9-stable'
+  gem "foreman_remote_execution", "< 13"
```

Comment:
Once https://github.com/Katello/katello/commit/4a91ec50461a5bbdac6929f7f4cb42eddb3a7afb is released this shouldn't be needed anymore, but for now this is ok.